src/components/Team/UpdateProfile.js

```diff
@@ -228,15 +242,15 @@ const UpdateProfile = props => {
 							{getFieldDecorator("name", {
 								rules: [
 									{
-										require: true,
+										required: true,
 										message: "Please enter your name!"
 									}
 								]
 							})(
 								<Input
 									type="text"
 									placeholder="Name"
-									onChange={e => setName(e.target.value)}
+									onChange={e => setNewName(e.target.value)}
```

Comment:
using ```onChange``` in these antd components is a wrong way to listen to changes. They will lose their values during state updates. Antd handles the listening for change events for us. You just need to grab the final values during submission.